doc/content/tutorials/KRUSTY.md

```diff
@@ -0,0 +1,885 @@
+| 614, instance   0 (of   1) | 50338 |     0 |     0 |     0 | 5.789e-04  |            |
+| 615, instance   0 (of   1) | 38227 |     0 |     0 |     0 | 2.942e-04  |            |
+| 616, instance   0 (of   1) | 34153 |     0 |     0 |     0 | 2.497e-04  |            |
+| 617, instance   0 (of   1) | 29815 |     0 |     0 |     0 | 6.205e-04  |            |
+| 618, instance   0 (of   1) | 24836 |     0 |     0 |     0 | 8.822e-04  |            |
+| 619, instance   0 (of   1) | 21965 |     0 |     0 |     0 | 1.981e-04  |            |
+| 620, instance   0 (of   1) | 21483 |     0 |     0 |     0 | 2.256e-04  |            |
+----------------------------------------------------------------------------------------
+
+ ===================>     AUXVARIABLES FOR OPENMC I/O     <===================
+
+      Subdomain:  subdomain name/ID
+    Temperature:  variable OpenMC reads temperature from (empty if no feedback)
+        Density:  variable OpenMC reads density from (empty if no feedback)
+
+-----------------------------------------
+|   Subdomain   | Temperature | Density |
+-----------------------------------------
+| gap_clamp     | temp        |         |
+| clamp         | temp        |         |
+| gap_vaccan    | temp        |         |
+| MLI           | temp        |         |
+| vacuum_can    | temp        |         |
+| gap_sleeve    | temp        |         |
+| sleeve        | temp        |         |
+| gap_ref       | temp        |         |
+| cavity_center | temp        |         |
+| ref_bottom    | temp        |         |
+| core          | temp        |         |
+| ref_top       | temp        |         |
+-----------------------------------------
+
+    Tally Score:  OpenMC tally score
+    AuxVariable:  variable where this score is written
+
+-------------------------------
+|  Tally Score  | AuxVariable |
+-------------------------------
+| kappa-fission | heat_source |
+-------------------------------
+
+ ===================>  OPENMC SUBDOMAIN MATERIAL MAPPING  <====================
+
+      Subdomain:  Subdomain name; if unnamed, we show the ID
+       Material:  OpenMC material name(s) in this subdomain; if unnamed, we
+                  show the ID. If N duplicate material names, we show the
+                  number in ( ).
+
+-----------------------------------
+|   Subdomain   |    Material     |
+-----------------------------------
+| gap_clamp     |  Helium for gap |
+| clamp         |  Haynes230      |
+| gap_vaccan    |  Helium for gap |
+| MLI           |  Mo Foil        |
+| vacuum_can    |  SS             |
+| gap_sleeve    |  Helium for gap |
+| sleeve        |  SS             |
+| gap_ref       |  Helium for gap |
+| cavity_center |  VOID           |
+| ref_bottom    |  BeO reflector  |
+| core          |  U-7.65Mo Fuel  |
+| ref_top       |  BeO reflector  |
+
+```
+
+This shows the OpenMC cells mapped to the MOOSE elements. The above message also shows the
```

Comment:
Is there a stochastic volume calculation? The last column in the table looks empty here